build-system/server/new-server/transforms/stories/stories-transform.ts

```diff
@@ -19,6 +19,14 @@ import {URL} from 'url';
 import {isValidScript} from '../utilities/script';
 import {CDNURLToLocalDistURL} from '../utilities/cdn';
 
+/**
+ * A list of options to correspond with options.json for testing purposes.
+ * To add an option, add the corresponding key-value pair into the 
+ * options.json, then add the field to this interface.
+ */
+interface OptionSet {
```

Comment:
lets go ahead and move this out to a common ts file so that people in the future will be inclined to use it. other than that. everything looks good to me now and i'll approve

Reply:
Moved to `utilities/option-set.ts`.